lib/fluent/plugin/in_tail.rb

```diff
@@ -451,6 +453,16 @@ def flush_buffer(tw)
             record[@path_key] ||= tw.path unless @path_key.nil?
             router.emit(tag, time, record)
           else
+            if @emit_unmatched_lines
+              record = { 'unmatched_line' => lb }
+              record[@path_key] ||= tail_watcher.path unless @path_key.nil?
+              tag = if @tag_prefix || @tag_suffix
+                      @tag_prefix + tw.tag + @tag_suffix
+                    else
+                      @tag
+                    end
+              router.emit(tag, Fluent::EventTime.now, record)
+            end
             log.warn "got incomplete line at shutdown from #{tw.path}: #{lb.inspect}"
```

Comment:
This line should be in `else`?

Reply:
I put it here because a similar code does the same thing https://github.com/fluent/fluentd/pull/2837/files#diff-1da710c9dcc8d0fc57996df7a9d39695R507.